my-sms/Gemfile

```diff
@@ -19,6 +19,7 @@ end
 group :development, :test do
   gem 'rspec-rails', '~> 3.6.0'
   gem 'factory_bot_rails', '~> 4.10.0'
+  gem 'test-unit'
```

Comment:
I didn't realise this was an actual dependency of rails 3.2, so my bad for suggesting you remove it in the first place.
They've removed it as a dependency in subsequent versions.

Reply:
Haha all good that was Matt's you commented on actually.

Reply:
Oh, man, I'm getting slightly mixed up in my reviews 😅